CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -313,6 +317,17 @@ local function isGuiVisible(gui, debug) -- true if any part of the gui is visibl
 	end
 end
 
+local function addHoverState(button, instance, applyNormal, applyHover)
+	local function indirectNormal()	applyNormal(instance) end
```

Comment:
The naming on these funcs are a bit confusing for me, I am not sure why it is called `indirectNormal` also I think `applyNormal` might want to be called `onNormalButtonState` or `onButtonNormalState` to disambiguate for the reader (remember 3d guis in VR have surface normals!)